.yarnrc.yml

```diff
@@ -8,21 +12,30 @@ logFilters:
 
 nodeLinker: node-modules
 
+npmAuditIgnoreAdvisories:
+  # Issue: yargs-parser Vulnerable to Prototype Pollution
+  # URL: https://github.com/advisories/GHSA-p9pc-299p-vxgp
+  # The affected version (<5.0.0) is only included via
+  # @ensdomains/ens via 'solc' which is not used in the imports
+  # we use from this package.
+  - 1088783
+  - 1089718 # GHSA-257v-vj4p-3w2h https://github.com/advisories/GHSA-257v-vj4p-3w2h
+  - '@material-ui/core (deprecation)'
+  - 'eth-sig-util (deprecation)'
```

Comment:
This is just a WIP at the moment I want to go through each audit failure and figure out if its something we should fix as part of this PR, as a follow up, etc, and then categorize each audit failure using # comments so that we know what we wanna do with them. 

Example "@material-ui/core (deprecation)" could go in the "dependency planned for removal" category